src/executables/csql.c

```diff
@@ -3384,6 +3384,7 @@ csql_connect (char *argument, CSQL_ARGUMENT * csql_arg)
   char *p = NULL;
   const char *err_msg;
   CSQL_ARGUMENT csql_new_arg;
+  char csql_db_name[DB_NAME_LEN];
```

Comment:
Since the use of this variable's address is outside the scope of this function, it is better to make it as a global variable.

Reply:
Right. If not, It may occur memory problem.
After I change it as global variable and then Can I merge this pr?

Reply:
Yes, I already approved this PR.

Reply:
Ok! Thank you!